clustered/client/src/main/java/org/ehcache/clustered/client/config/ClusteringServiceConfiguration.java

```diff
@@ -198,7 +198,7 @@ public Timeouts getTimeouts() {
    * @return the connection {@code Properties}
    */
   public Properties getProperties() {
-    return properties;
+    return new Properties(properties);
```

Comment:
This does not do what you believe it does :wink:

Also, you need to take a copy when assigning it to the field during construction to protect against mutation from the object you received.

Reply:
Done